robotbase/src/main/java/frc/robot/commands/intake/IntakeToShooterCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class IntakeToShooterCommand extends Command {
```

Comment:
We do not append the word `Command` to out commands. We have decided that doing this is unnecessary lego naming.